docs/misc.rst

```diff
@@ -0,0 +1,203 @@
+
+
+Notes
+-----
+
+Timing & The Microsoft Azure cloud
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Timing is a tricky subject when dealing with the Azure cloud as
+some operations can take seconds while others can take up to 20
+minutes in real-world deployments.
+
+Azure APIs make an effort to
+mitigate this by responding to asynchronous operations with a
+"retry after" number to indicate the number of seconds to wait
+before checking the status of an operation again.  Unfortunately,
+this number is more often a woeful underestimate and relying on it
+will result in a high percentage of Cloudify executions leading to
+failure because of too many task retries before an Azure resource
+reports it's successfully created or destroyed.
+
+For instance, creating a Resource Group is generally a synchronous
+operation as it almost always reports success immediately.  But,
+a Storage Account will almost never do this and will generally report
+that the operation was done asynchrnously and will tell the consumer to
+wait, say, 25 seconds before retrying.  By default, Cloudify limits the
+number of task retries to 11 which means that the Storage Account will
+need to report a successful operation with (25*11) 275 seconds (4.58 minutes).
+
+Storage Accounts can take up to 20 minutes to report a success and the default
+task retry limitations prevent this in many cases.  There are really only
+two options to mitigate this issue on the Cloudify side of things.
+
+ **#1** is to
+ take the reported "retry after" value from Azure with a grain of salt and
```

Comment:
~~take~~ the reported "retry after" value from Azure with ~~a grain of salt ~~ 

**consider** the reported "retry after" value from Azure with **skepticism **